### PR TITLE
Add Int64.fromString.

### PR DESCRIPTION
**Pull request by georgkoester**

### CODE REVIEW

**Comment by georgkoester** [comment]
Hi @Simn , I can do that. Proposal:
1. Add parseInt etc in Std and 
2. Add fromString on the Int/Float classes that reference the Std.parse implementations.
3. Tests

**Comment by Simn** [comment]
Let's just add `fromFloat` and `fromString` to Int64 for now. Modifying Std is always a bit of a mess.

**Comment by georgkoester** [comment]
Hi @waneck , do I need to add special treatment for the java/cs targets to make Int64 parsing/conversion work?

**Comment by Justinfront** [comment]
georgkoester I don't know if this class of mine is useful https://github.com/Justinfront/Jigsawx/blob/master/Int64.hx I probably have a few variations, but it seemed enough to get by with java swing graphics that often have Int64.

**Comment by georgkoester** [comment]
Thanks @Justinfront , I also like your parsing approach using the int values of the char codes directly (that's what you're talking about, right?) - but I propose the more readable solution. This is of course open for disucssion.

**Comment by Justinfront** [comment]
The readDigits code is borrowed from https://github.com/HaxeFoundation/haxe/blob/development/std/haxe/Unserializer.hx#L147 after some research back when Int64 support on Java was target was really limited, I presume it's fast no idea how it compares with current Int64 abstract stuff.

**Comment by waneck** [comment]
You just need to add the same implementation to the
std/java/_std/haxe/Int64.hx and the cs counterpart of it. The API is the
same

File: tests/unit/src/unit/TestInt64.hx [hunk 1]
```diff
@@ -112,4 +112,103 @@ class TestInt64 extends Test {
...
+		}
+
+		try
+		{
+			fromFloat(9007199254740992);
+			f(true);
+		}
+		catch (e:Dynamic)
+		{
+			// fine
```
Comment (Simn) [inline tests/unit/src/unit/TestInt64.hx]: For future reference, you can use `exc` for this kind of testing.

**Comment by Simn** [comment]
We have finally made the abstract Int64 change. Could you resubmit your PR?

**Pull request closed without merging**